config/routes.rb

```diff
@@ -7,7 +7,8 @@
       post 'auth/login', to: 'auth#create'
       resources :categories, only: %i[create update destroy]
       post '/organization/public', to: 'organizations#create'
-      resources :users, only: %i[:update index]
+      resources :slides, only: :index
+      resources :users, only: %i[:update index]      
```

Comment:
Muy bueno esto, agrego una pequeña sugerencia para respetar convenciones de rails:
```suggestion
      resources :users, only: %i[index update]      
```